app/src/main/java/com/procleus/brime/ui/SettingsActivity.java

```diff
@@ -69,6 +72,19 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
     }
 
+    public void signOut(){
+        SharedPreferences sharedpreferences = getSharedPreferences(SigninActivity.PREF, Context.MODE_PRIVATE);
+        SharedPreferences.Editor editor = sharedpreferences.edit();
+        editor.remove("emailpref");
```

Comment:
I would highly recommend to make a `AccountUtils` class which handles all the account related actions with the shared preferences.
for example 
```
public class AccountUtils {
    private static final String TAG = AccountUtils.class.getSimpleName();

    public static final String PREF_ACTIVE_ACCOUNT = "chosen_account";
    public static final String PREF_ACTIVE_ACCOUNT_NAME = "chosen_account_name";
    public static final String PREF_ACTIVE_ACCOUNT_EMAIL = "chosen_account_email";
   
    private static SharedPreferences getSharedPreferences(final Context context) {
        return PreferenceManager.getDefaultSharedPreferences(context);
    }

    /**
     * Specify whether the app has an active account set.
     *
     * @param context Context used to lookup {@link SharedPreferences} the value is stored with.
     */

    public static boolean hasActiveAccount(final Context context) {
        return !TextUtils.isEmpty(getActiveAccountEmail(context));
    }
   public static String getActiveAccountEmail(final Context context) {
        SharedPreferences sp = getSharedPreferences(context);
        return sp.getString(PREF_ACTIVE_ACCOUNT, null);
    }
    public static void setActiveAccount(final Context context, final String accountEmail) {
        SharedPreferences sp = getSharedPreferences(context);
        sp.edit().putString(PREF_ACTIVE_ACCOUNT, accountEmail).apply();
    }

    public static void clearActiveAccount(final Context context) {
        Log.d(TAG, "Clearing active account");
        SharedPreferences sp = getSharedPreferences(context);
        sp.edit().remove(PREF_ACTIVE_ACCOUNT).apply();
        sp.edit().remove(PREF_ACTIVE_ACCOUNT_NAME).apply();
        sp.edit().remove(PREF_ACTIVE_ACCOUNT_REFERRAL_CODE).apply();
    }
}
```